src/commands/txCommands/push.js

```diff
@@ -20,6 +20,8 @@ const pushResource$ = poData =>
 		.zip(gitBranch$, branchResourceExists$)
 		.flatMap(([gitBranch, branchResourceExists]) => {
 			if (Object.keys(poData).length) {
+				console.log('branchResourceExists: ', branchResourceExists);
```

Comment:
I would recommend making these logs more plain-language, e.g. `branch exists in transifex:` and `translation data`